kjdoc/tour.md

```diff
@@ -1021,6 +1021,19 @@ KJ provides an advanced, cross-platform filesystem API in `kj/filesystem.h`. Fea
 * Symlinks, hard links, listing directories, recursive delete, recursive create parents, recursive copy directory, memory mapping, and unnamed temporary files are all exposed and easy to use.
 * Sparse files ("hole punching"), copy-on-write file cloning (`FICLONE`, `FICLONERANGE`), `sendfile()`-based copying, `renameat2()` atomic replacements, and more will automatically be used when available.
 
+```c++
```

Comment:
This needs some kind of a preface I think before just dumping the code. Something like "Here are API examples for accessing files."

Reply:
Hmm, the rest of the tour uses a pretty similar format: https://github.com/capnproto/capnproto/blob/363cefdf46d37f399ba705ba9d9c93e7fa74bcbe/kjdoc/tour.md#unit-testing-tips
I can change it if you like, though.